### DISCUSSION TITLE
Can't Update to 2.5.219

### DISCUSSION THREAD

**Discussion by charm-games** [requarks/wiki#4695] [Error / Bug Report]
Trying to update from 2.5.170 to 2.5.219 and it keeps giving me an error

Error: getaddrinfo ENOTFOUND wiki-update-companion

Can someone offer some help pls?

Thanks

Dinos

**Comment by ieaster1** [comment]
Is your system behind a proxy or does it have unfettered egress?

**Comment by charm-games** [comment]
We're running it as a digital ocean droplet so I'm not sure but I imagine it is probably behind a proxy in that case.

**Reply by ieaster1** [reply]
I would only think 2 different possibilities.

1, the `wiki-update-companion` container isn't running.  You should check this from your system and review the [doc](https://docs.requarks.io/install/ubuntu#setup-containers). You'll need to create the container if your didn't.

2, Possible DNS issues reaching out to **registry-1.docker.io**

**Reply by NGPixel** [reply]
Make sure the `wiki-update-companion` container is running (check with `sudo docker ps -a`)
